gtsfm/data_association/data_assoc.py

```diff
@@ -72,7 +72,9 @@ def __validate_track(self, sfm_track: Optional[SfmTrack]) -> bool:
     def run(
         self,
         num_images: int,
-        cameras: Dict[int, gtsfm_types.CAMERA_TYPE],
+        wRi_list: List[Optional[Rot3]],
+        wti_list: List[Optional[Point3]],
```

Comment:
@akshay-krishnan @dellaert @ayushbaid I think this will conflict with the changes I've been working to bring in from Ayush's hilti branch, where TA now returns a Pose3, instead of a Point3:

https://github.com/borglab/gtsfm/pull/515/files#diff-57f230607899ca8b4f6072976b6f231b322a3c7e6107287f2eb5aa9798e9d52aR266

Can we defer this change until I can get those other changes merged? Currently gtsfm is not functional with gtsam latest because the TA interface is now out of date.

Reply:
@johnwlambert sorry about that, wasnt aware. It looks like the PR you point to is pretty big. I don't think its a good idea to block this small PR until a large PR is reviewed. 

If you'd like to submit the Pose3 change in a quick small PR, we can wait for it.

Reply:
No problem -- sure, let me put those proposed changes into a quick PR in the next day or two. Might simplify this PR as well. Thanks!